tests/wpt/web-platform-tests/cssom/CSSStyleSheet.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>CSSOM - CSSStyleSheet interface</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+        body { width: 50%; }
+        #foo { height: 100px; }
+    </style>
+
+    <script>
+    test(function () {
+        var styleSheet = document.styleSheets[0];
```

Comment:
Ensure that it's equal to `document.getElementById("my-stylesheet").style` (giving the `<style>` element an id above)